app/src/main/java/org/mozilla/tv/firefox/channels/ChannelConfig.kt

```diff
@@ -16,19 +18,25 @@ data class ChannelConfig(
     val onClickTelemetry: ((ChannelTile) -> Unit)? = null,
     val onLongClickTelemetry: ((ChannelTile) -> Unit)? = null,
     val onFocusTelemetry: ((ChannelTile, Boolean) -> Unit)? = null,
-    val itemsMayBeRemoved: Boolean = false
+    val itemsMayBeRemoved: Boolean = false,
+    val isEnabledInCurrentExperiment: Boolean,
+    val enabledInLocales: KillswitchLocales
 ) {
     companion object {
         fun getPocketConfig(): ChannelConfig = ChannelConfig(
                 onClickTelemetry = { tile -> TELEMETRY.pocketVideoClickEvent(tile.id) },
                 // TODO focus telemetry should probably only be sent on focus gain, but this is
                 //  how our previous implementation worked. Keeping this to maintain data consistency
-                onFocusTelemetry = { tile, _ -> TELEMETRY.pocketVideoImpressionEvent(tile.id) }
+                onFocusTelemetry = { tile, _ -> TELEMETRY.pocketVideoImpressionEvent(tile.id) },
+                isEnabledInCurrentExperiment = true,
+                enabledInLocales = KillswitchLocales.ActiveIn(Locale.US)
```

Comment:
I think there's a redundant check here:
- `currentLanguageIsEnglish` is still used in the code as a dependency to Pocket code
- We are enabling a killswitch here.

As I note elsewhere, Pocket should be enabled for all english locales, not just Locale.US

Reply:
Hm.  We will need to plan how to move `currentLanguageIsEnglish` from other code.  It was added to avoid sending Pocket requests that would never be shown to the user.  I think reworking that is out of scope for this story.

Reply:
Can you file a follow-up bug? If it needs to get done this sprint, please raise it.

Reply:
No, it works as is, but it's duplicated logic and a landmine for future changes.  We definitely don't need to block on it, which I think would make that issue an aspirational refactor.  I can add a TODO in code though.